fuel-block-producer/src/config.rs

```diff
@@ -1,20 +1,6 @@
-use fuel_core_interfaces::common::{
-    fuel_tx::ConsensusParameters,
-    fuel_types::Word,
-};
+use fuel_core_interfaces::common::fuel_tx::ConsensusParameters;
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 pub struct Config {
-    pub max_gas_per_block: Word,
     pub consensus_params: ConsensusParameters,
 }
-
-impl Default for Config {
-    fn default() -> Self {
-        Self {
-            // TODO: pick a reasonable default value based on gas scheduling analysis
```

Comment:
Was this TODO fixed / survived somewhere else?

Reply:
It was moved to be an input argument to the trigger, see fuel-poa-coordinator/src/config.rs